README.md

```diff
@@ -102,20 +96,42 @@ for path in battery_paths:
 ```
 
 These paths are unfortunately non-deterministic and will change on each
-iteration.
+iteration. For long running processes please prefer to keep an open `Battery` object and open + close on read errors.
```

Comment:
Want to link to the issue/comment with good pseudocode? Alternately can include pseudocode directly in the README.

Reply:
Will add code.